packages/app/models/service.ts

```diff
@@ -59,6 +59,10 @@ export async function getModel(
             // execute the macro templates for this model and get their values
             const [error, populatedTemplates] = await runModelTemplates(model)
 
+            if (error) {
```

Comment:
Broken model templates were not properly handling errors, so we now re-raise them.